touchview/src/main/java/com/ortiz/touchview/TouchImageView.java

```diff
@@ -1543,8 +1543,7 @@ public void run() {
          * @return progress of the interpolation
          */
         private float interpolate() {
-            long currTime = System.currentTimeMillis();
-            float elapsed = (currTime - startTime) / zoomTime;
+            float elapsed = (System.currentTimeMillis() - startTime) / (float) zoomTimeMillis;
```

Comment:
@arttttt  
I guess, staying on `int` ,and only use it here as float for the calculation result, makes it more clear to be milliseconds

Reply:
I just wanted to write about it in my PR